Resources/Prototypes/Catalog/VendingMachines/Inventories/sec.yml

```
@@ -20,7 +20,8 @@
     RiotShield: 2
     RiotLaserShield: 2
     RiotBulletShield: 2
-    RadioHandheldSecurity: 5
+    RadioHandheldExternalAdministrativeZoneSecurityService: 5 # Fire-Station
+    RadioHandheldHeavyContainmentSecurityService: 5 # Fire-Station
```

```suggestion
    RadioHandheldExternalAdministrativeZoneSecurityService: 5 # Fire added
    RadioHandheldHeavyContainmentSecurityService: 5 # Fire added
```

Комментарии в одном стиле должны быть